src/django_htmx/management/commands/downloadhtmx.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import os
+from pathlib import Path
+from typing import TYPE_CHECKING
+from typing import Any
+from urllib.parse import urlparse
+from urllib.parse import urlunparse
+from urllib.request import urlopen
+
+from django.conf import settings
+from django.core.checks import Tags
+from django.core.management.base import BaseCommand
+from django.core.management.base import CommandError
+
+if TYPE_CHECKING:
+    from argparse import ArgumentParser
+
+
+class Command(BaseCommand):
+    help = "Download the given htmx version and the extensions."
+    requires_system_checks = [Tags.staticfiles]
```

Comment:
Why did you do this? Were you trying to [speed up the checks](https://adamj.eu/tech/2024/03/23/django-optimizing-system-checks/) ?
```suggestion
```

Reply:
I was thinking that since we use the `STATICFILES_DIRS`, it makes sense to at least check if the finders are configured correctly. But I agree with the other comment of yours about not having this default, so it's not needed after all

Reply:
Done in 8a2a20e08a1c154e58cc375b032039495df3f646